tests/util/mod.rs

```diff
@@ -199,6 +199,11 @@ impl<'a> TestManager<'a> {
 
         Ok(())
     }
+
+    /// Remove a temporary table from the list of temporary tables.
+    pub fn remove_temporary_table(&mut self, table_name: String) {
+        self.temporary_tables.retain(|s| s != &table_name);
```

Comment:
nit: You can improve the interface using `AsRef<str>`. But, it is not necessary. 

```suggestion
    pub fn remove_temporary_table<S: AsRef<str>>(&mut self, table_name: S) {
        let table_name = table_name.as_ref();
        self.temporary_tables.retain(|s| s != table_name);
```